lib/functions-misc.php

```diff
@@ -48,6 +48,26 @@ function clean_site_blog_info() {
 }
 add_action( 'after_setup_theme', 'clean_site_blog_info' );
 
+
+// Show exhibitor year column
+add_filter( 'manage_exhibitor_posts_columns', 'add_exhibitor_year_column' );
```

Comment:
wassup with this syntax. surely the filter/action hooks should sit next to the function as per convention?